model/activist.go

```diff
@@ -441,6 +444,18 @@ type GetActivistOptions struct {
 	UpcomingFollowupsOnly bool   `json:"upcoming_followups_only"`
 }
 
+type GeocodeResponse struct {
```

Comment:
please a comment indicating that this structure corresponds to Google's geo coding API with a link to that API's documentation